[E2E-SIM] 70 failures — 2026-03-21T01:50:47.092Z

## E2E Simulation Failure Report

**Run ID**: `7f907c09-4693-4468-b102-3d710e4ff0bc`
**Failed Checks**: 70/200
**Environment**: development

## Failed Assertions
- **verification_approved**: expected `"approved+pvu"`, got `"approved"` — column partner_verified_users.id does not exist
- **verification_approved**: expected `"approved+pvu"`, got `"approved"` — column partner_verified_users.id does not exist
- **verification_approved**: expected `"approved+pvu"`, got `"approved"` — column partner_verified_users.id does not exist
- **verification_approved**: expected `"approved+pvu"`, got `"approved"` — column partner_verified_users.id does not exist
- **verification_approved**: expected `"approved+pvu"`, got `"approved"` — column partner_verified_users.id does not exist
- **verification_approved**: expected `"approved+pvu"`, got `"approved"` — column partner_verified_users.id does not exist
- **verification_approved**: expected `"approved+pvu"`, got `"approved"` — column partner_verified_users.id does not exist
- **verification_approved**: expected `"approved+pvu"`, got `"approved"` — column partner_verified_users.id does not exist
- **verification_approved**: expected `"approved+pvu"`, got `"approved"` — column partner_verified_users.id does not exist
- **verification_approved**: expected `"approved+pvu"`, got `"approved"` — column partner_verified_users.id does not exist
- **verification_approved**: expected `"approved+pvu"`, got `"approved"` — column partner_verified_users.id does not exist
- **verification_approved**: expected `"approved+pvu"`, got `"approved"` — column partner_verified_users.id does not exist
- **verification_approved**: expected `"approved+pvu"`, got `"approved"` — column partner_verified_users.id does not exist
- **verification_approved**: expected `"approved+pvu"`, got `"approved"` — column partner_verified_users.id does not exist
- **verification_approved**: expected `"approved+pvu"`, got `"approved"` — column partner_verified_users.id does not exist
- **verification_approved**: expected `"approved+pvu"`, got `"approved"` — column partner_verified_users.id does not exist
- **verification_approved**: expected `"approved+pvu"`, got `"approved"` — column partner_verified_users.id does not exist
- **verification_approved**: expected `"approved+pvu"`, got `"approved"` — column partner_verified_users.id does not exist
- **verification_approved**: expected `"approved+pvu"`, got `"approved"` — column partner_verified_users.id does not exist
- **verification_approved**: expected `"approved+pvu"`, got `"approved"` — column partner_verified_users.id does not exist
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **refund_processed**: expected `"completed|requested"`, got `"failed"` — refund_status is 'failed'
- **settlement_ready**: expected `"READY"`, got `"PENDING"` — status is 'PENDING'
- **settlement_ready**: expected `"READY"`, got `"PENDING"` — status is 'PENDING'
- **settlement_ready**: expected `"READY"`, got `"PENDING"` — status is 'PENDING'
- **settlement_ready**: expected `"READY"`, got `"PENDING"` — status is 'PENDING'
- **settlement_ready**: expected `"READY"`, got `"PENDING"` — status is 'PENDING'
- **settlement_ready**: expected `"READY"`, got `"PENDING"` — status is 'PENDING'
- **settlement_ready**: expected `"READY"`, got `"PENDING"` — status is 'PENDING'
- **settlement_ready**: expected `"READY"`, got `"PENDING"` — status is 'PENDING'
- **settlement_ready**: expected `"READY"`, got `"PENDING"` — status is 'PENDING'
- **settlement_ready**: expected `"READY"`, got `"PENDING"` — status is 'PENDING'
- **settlement_ready**: expected `"READY"`, got `"PENDING"` — status is 'PENDING'
- **settlement_ready**: expected `"READY"`, got `"PENDING"` — status is 'PENDING'
- **settlement_ready**: expected `"READY"`, got `"PENDING"` — status is 'PENDING'
- **settlement_ready**: expected `"READY"`, got `"PENDING"` — status is 'PENDING'
- **settlement_ready**: expected `"READY"`, got `"PENDING"` — status is 'PENDING'
- **settlement_ready**: expected `"READY"`, got `"PENDING"` — status is 'PENDING'
- **settlement_ready**: expected `"READY"`, got `"PENDING"` — status is 'PENDING'
- **settlement_ready**: expected `"READY"`, got `"PENDING"` — status is 'PENDING'
- **settlement_ready**: expected `"READY"`, got `"PENDING"` — status is 'PENDING'
- **settlement_ready**: expected `"READY"`, got `"PENDING"` — status is 'PENDING'

## Axiom 디버깅
Edge Function 로그는 Axiom에서 확인할 수 있습니다.

```
axiom query "['edge-functions'] | where metadata.runId == '7f907c09-4693-4468-b102-3d710e4ff0bc'"
```

또는 Axiom 대시보드에서 dataset `edge-functions`를 선택하고 `metadata.runId == '7f907c09-4693-4468-b102-3d710e4ff0bc'`로 필터링하세요.

## Log Details
No storage log available

<details>
<summary>📋 Full Log (truncated to 60KB)</summary>

```log
[2026-03-21T01:49:59.803Z] [INFO] [create:party_created] Created: [E2E] 대학생 밍글
{"partyId":"8b8dd952-e087-45d1-aeca-1e653639962e"}
[2026-03-21T01:50:00.427Z] [INFO] [create:party_created] Created: [E2E] 직장인 밍글
{"partyId":"f974828f-b502-4421-a117-8a1f03cd8cc2"}
[2026-03-21T01:50:01.029Z] [INFO] [create:party_created] Created: [E2E] 소셜 파티
{"partyId":"a988b790-cfd0-41e5-83e1-8ec4a386d697"}
[2026-03-21T01:50:01.577Z] [INFO] [create:party_created] Created: [E2E] 주말 밍글
{"partyId":"bac8fa93-285f-465a-8fa3-09dfeaa2dc1b"}
[2026-03-21T01:50:02.166Z] [INFO] [create:party_created] Created: [E2E] 네트워킹
{"partyId":"ad9e7621-2056-4f9a-aeec-a341c3bd4a9a"}
[2026-03-21T01:50:02.167Z] [INFO] [create:done] Created 5 parties, 20 events
[2026-03-21T01:50:03.012Z] [INFO] [apply:event_applied] Event c3737d47-dbd4-499f-984d-6887847dab4b: 6 applications
[2026-03-21T01:50:03.501Z] [INFO] [apply:event_applied] Event 0bc94ce7-d4b2-43ed-b0b4-3941b79ade4a: 6 applications
[2026-03-21T01:50:03.931Z] [INFO] [apply:event_applied] Event 0b82a545-5823-4b6d-9b9c-aed49c13f3c1: 6 applications
[2026-03-21T01:50:04.324Z] [INFO] [apply:event_applied] Event f09c933b-81f7-4d26-b6a4-56825336361f: 6 applications
[2026-03-21T01:50:04.719Z] [INFO] [apply:event_applied] Event 7b09f85e-7899-41df-9849-8ca2c03024c3: 6 applications
[2026-03-21T01:50:05.202Z] [INFO] [apply:event_applied] Event 14417c17-5d49-4c84-be20-6ffd13439ba2: 6 applications
[2026-03-21T01:50:05.653Z] [INFO] [apply:event_applied] Event 69b09812-8a04-438f-964a-764a5183a012: 6 applications
[2026-03-21T01:50:06.081Z] [INFO] [apply:event_applied] Event 0a48c33e-fb7d-4708-9478-d576b81ddedf: 6 applications
[2026-03-21T01:50:06.563Z] [INFO] [apply:event_applied] Event 490a8877-d369-47e7-b9e6-8974502c099f: 6 applications
[2026-03-21T01:50:06.981Z] [INFO] [apply:event_applied] Event 89fcc84c-630e-40ca-a936-dde448c43c63: 6 applications
[2026-03-21T01:50:07.404Z] [INFO] [apply:event_applied] Event 1b575805-72b6-4835-8f3e-d13f61907d89: 6 applications
[2026-03-21T01:50:07.843Z] [INFO] [apply:event_applied] Event 24ae1775-6420-4269-944e-4a802ce449de: 6 applications
[2026-03-21T01:50:08.264Z] [INFO] [apply:event_applied] Event 7ad4da22-ff93-4307-bfb1-08aae6ef2048: 6 applications
[2026-03-21T01:50:08.760Z] [INFO] [apply:event_applied] Event 3cf9e99a-b799-4be1-9ee5-e73e49bfda14: 6 applications
[2026-03-21T01:50:09.190Z] [INFO] [apply:event_applied] Event c18d4b29-4d86-439f-ac4e-23daf2324d19: 6 applications
[2026-03-21T01:50:09.666Z] [INFO] [apply:event_applied] Event 1f3ad404-9826-433c-97ed-53775f2a3941: 6 applications
[2026-03-21T01:50:10.083Z] [INFO] [apply:event_applied] Event 3200a102-66da-4e44-a245-9b2a93e744f7: 6 applications
[2026-03-21T01:50:10.581Z] [INFO] [apply:event_applied] Event 3c2afcad-14ef-4276-a27a-92211ed52cbd: 6 applications
[2026-03-21T01:50:11.043Z] [INFO] [apply:event_applied] Event 55848c6a-b35a-4269-97ef-955edb76e194: 6 applications
[2026-03-21T01:50:11.446Z] [INFO] [apply:event_applied] Event dcdfe0ab-8eab-41a5-a4ba-90f27c12b653: 6 applications
[2026-03-21T01:50:11.873Z] [INFO] [apply:event_applied] Event f3833ebb-d9dd-4e7b-8623-e33a0de32fdf: 6 applications
[2026-03-21T01:50:12.271Z] [INFO] [apply:event_applied] Event 167237ff-eacd-492c-8fa8-cc98815f156b: 6 applications
[2026-03-21T01:50:12.740Z] [INFO] [apply:event_applied] Event c4ff0fa9-8ddb-4427-8022-7a14f4810512: 6 applications
[2026-03-21T01:50:13.198Z] [INFO] [apply:event_applied] Event c9316c9c-c5d2-4701-a031-70cbd5aea6a6: 6 applications
[2026-03-21T01:50:13.647Z] [INFO] [apply:event_applied] Event 415fedf4-e448-4678-86bb-11988a0d9f1d: 6 applications
[2026-03-21T01:50:14.062Z] [INFO] [apply:event_applied] Event 9a346203-37eb-4a87-b07a-fcdfe3718e7e: 6 applications
[2026-03-21T01:50:14.514Z] [INFO] [apply:event_applied] Event db1766ed-9ab0-408e-b997-dc3ae37dc426: 6 applications
[2026-03-21T01:50:14.995Z] [INFO] [apply:event_applied] Event f7c785dd-adb4-4978-91d2-e40f5536d763: 6 applications
[2026-03-21T01:50:15.396Z] [INFO] [apply:event_applied] Event 6d6f87d9-013b-4152-bb9f-b3b719d1b873: 6 applications
[2026-03-21T01:50:15.804Z] [INFO] [apply:event_applied] Event de3e6b3a-6fdb-4e93-8371-ca21343f23d1: 6 applications
[2026-03-21T01:50:16.210Z] [INFO] [apply:event_applied] Event 317f45f1-962e-4b11-943f-e344869bc2c7: 6 applications
[2026-03-21T01:50:16.654Z] [INFO] [apply:event_applied] Event b869aedf-d5df-4ccb-8eff-765d5432aaba: 6 applications
[2026-03-21T01:50:17.055Z] [INFO] [apply:event_applied] Event 2b2fcf7a-0341-455e-b53e-77cc780f129d: 6 applications
[2026-03-21T01:50:17.489Z] [INFO] [apply:event_applied] Event d8b5f308-4a17-4bc3-a25d-8542a24e2519: 6 applications
[2026-03-21T01:50:17.860Z] [INFO] [apply:event_applied] Event 283c1fe3-1302-48c6-b4e0-4b82e44d256b: 6 applications
[2026-03-21T01:50:18.321Z] [INFO] [apply:event_applied] Event 2541600c-27b0-4cdf-b165-caabd3219806: 6 applications
[2026-03-21T01:50:18.670Z] [INFO] [apply:event_applied] Event 56bd8d7a-b330-4022-ae33-09a904b80cf9: 6 applications
[2026-03-21T01:50:19.104Z] [INFO] [apply:event_applied] Event 418aed2b-8aa2-4cc5-8d73-cea5a64630fa: 6 applications
[2026-03-21T01:50:19.540Z] [INFO] [apply:event_applied] Event 4add1131-3097-44d5-8704-c0478b70c591: 6 applications
[2026-03-21T01:50:19.968Z] [INFO] [apply:event_applied] Event 2c0cc5b0-935f-4548-a7a9-5241daf22324: 6 applications
[2026-03-21T01:50:19.968Z] [INFO] [apply:done] Total: 240 (196 paid, 44 pending_review)
[2026-03-21T01:50:19.968Z] [INFO] [approve:split] Processing 44 apps: 35 approve, 9 reject
[2026-03-21T01:50:20.150Z] [INFO] [approve:approved] App 2b6af5f6-24a8-4aff-a170-a26a079e1e9c directly approved (no verification required)
[2026-03-21T01:50:20.295Z] [INFO] [approve:approved] App f5a5801a-1b96-4ea6-b778-18edc9742d71 directly approved (no verification required)
[2026-03-21T01:50:20.452Z] [INFO] [approve:approved] App 42c997ac-2def-484f-a7fd-e0d3708e5444 directly approved (no verification required)
[2026-03-21T01:50:20.602Z] [INFO] [approve:approved] App c89dfdf1-bfd9-4b0b-a376-01519b22a96c directly approved (no verification required)
[2026-03-21T01:50:20.749Z] [INFO] [approve:approved] App dab9fcc3-4d45-49a2-9178-a86a23f6cf8c directly approved (no verification required)
[2026-03-21T01:50:20.898Z] [INFO] [approve:approved] App 9eb57309-cc3e-4071-bfe9-644776f240a3 directly approved (no verification required)
[2026-03-21T01:50:21.041Z] [INFO] [approve:approved] App f23c1883-c269-4ae3-ab41-4235e52c8f0c directly approved (no verification required)
[2026-03-21T01:50:21.188Z] [INFO] [approve:approved] App fec5c81f-9055-44c7-aa07-16631ce59c06 directly approved (no verification required)
[2026-03-21T01:50:21.336Z] [INFO] [approve:approved] App b308e41b-fe97-4ec5-8c5a-39104114afae directly approved (no verification required)
[2026-03-21T01:50:21.506Z] [INFO] [approve:approved] App f1112fb5-36a5-4c6e-bdbc-fedfbe73d78d directly approved (no verification required)
[2026-03-21T01:50:21.678Z] [INFO] [approve:approved] App 6751b78d-40b3-4f95-8413-b50b25f5d40e directly approved (no verification required)
[2026-03-21T01:50:21.825Z] [INFO] [approve:approved] App e3f1bb4e-db4d-43cf-afa1-201580b50716 directly approved (no verification required)
[2026-03-21T01:50:22.147Z] [INFO] [approve:approved] App f843caf5-4b93-459e-9a78-d34e8f0f1bd1 approved via verification 880d129d-6331-47bd-b3ac-571a33f79891
[2026-03-21T01:50:22.412Z] [INFO] [approve:approved] App 64ad9a84-946b-4101-ad97-a8695050ebd3 approved via verification dbdfcce1-a29d-4dc0-b573-1f3d22623a13
[2026-03-21T01:50:22.569Z] [INFO] [approve:approved] App 7a9fdeb6-7ebb-48cc-afa0-a03d7e20e775 directly approved (no verification required)
[2026-03-21T01:50:22.732Z] [INFO] [approve:approved] App 55e21141-51f8-4ff2-948a-0aedc2be1c07 directly approved (no verification required)
[2026-03-21T01:50:22.947Z] [INFO] [approve:approved] App 7c93b961-501f-4338-a5e7-39e29d035c94 approved via verification dd6dbab5-4f57-4751-9abd-bb3298e1f5a5
[2026-03-21T01:50:23.170Z] [INFO] [approve:approved] App 7bb37251-b1f2-4eaf-b3f4-383b7b69f345 approved via verification 688116e8-7fb3-460e-b16a-17b252f7cb23
[2026-03-21T01:50:23.384Z] [INFO] [approve:approved] App 797d09a6-305b-49d4-96bc-6718af53056a approved via verification c05878e8-398a-4979-b2e8-0c116529c7ae
[2026-03-21T01:50:23.586Z] [INFO] [approve:approved] App 69666cc5-e1ab-480f-83f4-7eda836210dc approved via verification d8a4ad3b-e3af-497c-aaf9-2d696b354078
[2026-03-21T01:50:23.791Z] [INFO] [approve:approved] App 04a9d435-c36c-47fe-9eae-bfb6af36fd40 approved via verification 84247e37-2923-4471-8dd5-05b81a50a0d6
[2026-03-21T01:50:24.005Z] [INFO] [approve:approved] App bb86a0d6-34c1-4c45-aed7-69b849c655b6 approved via verification 1a13b1e3-91ff-4ffa-81eb-e249a15478a8
[2026-03-21T01:50:24.214Z] [INFO] [approve:approved] App 73a6ff03-6b3a-4db8-90ae-7b266e8b3fe1 approved via verification 38854f56-78cd-493c-8499-a8f011876af2
[2026-03-21T01:50:24.424Z] [INFO] [approve:approved] App 1559ff50-e484-4296-883a-7d5d0fe66b2c approved via verification f6eaeb1d-05e9-4050-aa7a-de789dc3230a
[2026-03-21T01:50:25.055Z] [INFO] [approve:approved] App 7928f790-806d-49b0-bb05-3250d35759f9 directly approved (no verification required)
[2026-03-21T01:50:25.281Z] [INFO] [approve:approved] App eae3e3ed-19dc-4221-b770-c2cbf8e15c33 approved via verification 9fc3880f-1980-4edb-85af-aff868cb82be
[2026-03-21T01:50:25.495Z] [INFO] [approve:approved] App 7f838c8d-3063-48fa-af5c-59330c19f930 approved via verification 43b431b3-241b-43c6-9150-cf4960906bc2
[2026-03-21T01:50:25.715Z] [INFO] [approve:approved] App fbdc062a-76c4-419e-a190-d40e5727d737 approved via verification 730b490c-0f58-4014-8e9d-b226080c8cce
[2026-03-21T01:50:25.923Z] [INFO] [approve:approved] App 867c1f50-d1d7-4c97-8fe0-5ca85030284f approved via verification 4e1c9b3b-a282-4bff-83b7-534c7c2c2984
[2026-03-21T01:50:26.138Z] [INFO] [approve:approved] App eb3c5e50-4d41-4209-958f-f1a90cdbf58d approved via verification 288fcfbb-0f63-454d-85e7-4e7ea4f49838
[2026-03-21T01:50:26.355Z] [INFO] [approve:approved] App 9623ba30-e12e-4f88-80ac-fb2e0c2bd342 approved via verification 2eb2809e-374e-4bc3-a4b9-3d5aee1bf104
[2026-03-21T01:50:26.562Z] [INFO] [approve:approved] App 0e15f112-cf9f-45a1-b104-cf80567730b9 approved via verification c55f182c-fec3-4c57-a919-5e78c84c6390
[2026-03-21T01:50:26.774Z] [INFO] [approve:approved] App c701ea42-245d-4801-a9dd-8a9f4c13fa65 approved via verification 6ff65c59-197b-4d74-a818-c00d764a5ca3
[2026-03-21T01:50:26.989Z] [INFO] [approve:approved] App 27afed5e-61a0-4e30-8afb-911987a94fce approved via verification b6352a1e-ad2b-4306-99ab-508cf64fc149
[2026-03-21T01:50:27.218Z] [INFO] [approve:approved] App 2505cebf-75cc-471f-af6d-e57b2356fe0a approved via verification 7530ebf4-16d8-4299-86be-aa49a83570a1
[2026-03-21T01:50:27.364Z] [INFO] [approve:rejected] App ce564188-2283-4e96-8053-ac321d8f6a90 rejected via verification 5ad1cd7e-f7a0-4d6e-8ef5-67f93fa0dffa
[2026-03-21T01:50:27.515Z] [INFO] [approve:rejected] App af38df3a-6685-4058-a385-26e78f0ef1e0 rejected via verification 243f5c43-f038-4789-97e7-6b9cbd443bcf
[2026-03-21T01:50:27.670Z] [INFO] [approve:rejected] App d4dea63b-bde1-4ba7-8525-d086ee72bfee rejected via verification 1af25a4f-4d25-4252-a821-c3a5605603fb
[2026-03-21T01:50:27.823Z] [INFO] [approve:rejected] App 45796dc4-acc1-4d80-86b4-973330d2c1ba rejected via verification b8f8687d-c5b0-4fc8-b36e-c9a0411a5a1b
[2026-03-21T01:50:27.967Z] [INFO] [approve:rejected] App 784cfe2d-bd3c-44ab-8eaf-523547ab1690 rejected via verification 3e8e68b6-6208-40e1-85fd-f761b6576a2e
[2026-03-21T01:50:28.119Z] [INFO] [approve:rejected] App ca75cf4c-1a3b-43f4-8a42-d9ef85dd4e12 rejected via verification 8bddb3fb-78e7-4030-ad29-27557840c80b
[2026-03-21T01:50:28.283Z] [INFO] [approve:rejected] App 6390380c-24fc-43a8-ae4d-2e441d954bd4 rejected via verification a22fe91c-2e31-4e06-9460-1fc1395c9323
[2026-03-21T01:50:28.438Z] [INFO] [approve:rejected] App 867e66d8-7084-4010-af66-3c20be7ba65f rejected via verification 512b33e4-22e9-4e85-b240-79b67bdbfb27
[2026-03-21T01:50:28.586Z] [INFO] [approve:rejected] App 3d7d5ab2-89d9-4cf8-bfd2-0c2e49a87042 rejected via verification 3f5b32e5-3555-4bfd-8302-8db686b661ba
[2026-03-21T01:50:28.586Z] [INFO] [approve:done] Phase 3 complete: 35 approved, 9 rejected
{"approvedCount":35,"rejectedCount":9}
[2026-03-21T01:50:28.587Z] [INFO] [refund:start] Processing 39 refund(s) out of 196 paid applications
{"refundCount":39,"totalPaid":196,"refundRate":0.2}
[2026-03-21T01:50:28.741Z] [INFO] [refund:refunded] App d58fd9e3-a2df-4774-aea2-abe10619461a refunded: 100% (20000/20000)
{"appId":"d58fd9e3-a2df-4774-aea2-abe10619461a","refund_percentage":100,"refund_amount":20000,"fee_amount":0,"refund_status":"completed"}
[2026-03-21T01:50:28.887Z] [INFO] [refund:refunded] App f4fae36a-a23e-49fd-abdf-f0374ad3f0cc refunded: 100% (20000/20000)
{"appId":"f4fae36a-a23e-49fd-abdf-f0374ad3f0cc","refund_percentage":100,"refund_amount":20000,"fee_amount":0,"refund_status":"completed"}
[2026-03-21T01:50:29.077Z] [INFO] [refund:refunded] App da628be0-055a-467e-9a5b-2924085a94a3 refunded: 100% (20000/20000)
{"appId":"da628be0-055a-467e-9a5b-2924085a94a3","refund_percentage":100,"refund_amount":20000,"fee_amount":0,"refund_status":"completed"}
[2026-03-21T01:50:29.242Z] [INFO] [refund:refunded] App 72eff9e5-3748-4f16-8d70-963979dc9a74 refunded: 100% (20000/20000)
{"appId":"72eff9e5-3748-4f16-8d70-963979dc9a74","refund_percentage":100,"refund_amount":20000,"fee_amount":0,"refund_status":"completed"}
[2026-03-21T01:50:29.394Z] [INFO] [refund:refunded] App 881ef466-42ee-4771-82e6-688bd2179fbc refunded: 100% (20000/20000)
{"appId":"881ef466-42ee-4771-82e6-688bd2179fbc","refund_percentage":100,"refund_amount":20000,"fee_amount":0,"refund_status":"completed"}
[2026-03-21T01:50:29.543Z] [WARN] [refund:assert_failed] Refund assertion failed for app d442e6cc-2395-41e4-b43c-323752211e84: refund_status is 'failed'
[2026-03-21T01:50:29.698Z] [WARN] [refund:assert_failed] Refund assertion failed for app 89c90d1f-600f-49ab-841f-6853b88ef4b9: refund_status is 'failed'
[2026-03-21T01:50:29.870Z] [WARN] [refund:assert_failed] Refund assertion failed for app 735371f9-14ed-491f-b4e9-7ce51f4b7c4b: refund_status is 'failed'
[2026-03-21T01:50:30.022Z] [WARN] [refund:assert_failed] Refund assertion failed for app 792f7558-2b2b-42c1-8c99-a19c12d0529f: refund_status is 'failed'
[2026-03-21T01:50:30.169Z] [WARN] [refund:assert_failed] Refund assertion failed for app 08d1e15b-597e-4190-95a8-15ece2de5876: refund_status is 'failed'
[2026-03-21T01:50:30.321Z] [WARN] [refund:assert_failed] Refund assertion failed for app 83be8095-38be-4763-ba0b-40adc455b3c3: refund_status is 'failed'
[2026-03-21T01:50:30.465Z] [WARN] [refund:assert_failed] Refund assertion failed for app 2e722610-7c21-4f32-bd6e-60af578a8cf0: refund_status is 'failed'
[2026-03-21T01:50:30.614Z] [WARN] [refund:assert_failed] Refund assertion failed for app 77002355-bc2f-4c04-85a1-3d7d93a8de26: refund_status is 'failed'
[2026-03-21T01:50:30.777Z] [WARN] [refund:assert_failed] Refund assertion failed for app c60cff3f-49ca-451f-ae88-6c23c7a11c56: refund_status is 'failed'
[2026-03-21T01:50:30.939Z] [WARN] [refund:assert_failed] Refund assertion failed for app f90f8ad6-2b8d-4968-8136-c4ef2c8702a4: refund_status is 'failed'
[2026-03-21T01:50:31.100Z] [WARN] [refund:assert_failed] Refund assertion failed for app cb07a8dd-fe44-4ef5-9091-eff6ca0b50c2: refund_status is 'failed'
[2026-03-21T01:50:31.252Z] [WARN] [refund:assert_failed] Refund assertion failed for app d5d56b5b-2a0d-4f0c-981b-267de2e75387: refund_status is 'failed'
[2026-03-21T01:50:31.395Z] [WARN] [refund:assert_failed] Refund assertion failed for app db7e7182-06db-4c27-86bd-4a7a4241c595: refund_status is 'failed'
[2026-03-21T01:50:31.541Z] [WARN] [refund:assert_failed] Refund assertion failed for app 92db7207-2715-4c15-bb7e-417bcadbcaa0: refund_status is 'failed'
[2026-03-21T01:50:31.701Z] [INFO] [refund:refunded] App 1f465277-44d0-4c78-b1ea-1a4110495656 refunded: 100% (20000/20000)
{"appId":"1f465277-44d0-4c78-b1ea-1a4110495656","refund_percentage":100,"refund_amount":20000,"fee_amount":0,"refund_status":"completed"}
[2026-03-21T01:50:31.845Z] [INFO] [refund:refunded] App 0f729790-b130-40f4-93f9-2eed0cd7eed5 refunded: 100% (20000/20000)
{"appId":"0f729790-b130-40f4-93f9-2eed0cd7eed5","refund_percentage":100,"refund_amount":20000,"fee_amount":0,"refund_status":"completed"}
[2026-03-21T01:50:31.999Z] [INFO] [refund:refunded] App 5ec6dbf4-1d99-4ab8-85d0-776283d681c4 refunded: 100% (20000/20000)
{"appId":"5ec6dbf4-1d99-4ab8-85d0-776283d681c4","refund_percentage":100,"refund_amount":20000,"fee_amount":0,"refund_status":"completed"}
[2026-03-21T01:50:32.143Z] [INFO] [refund:refunded] App 0347313b-78cf-42dd-97a4-06d85bf00e83 refunded: 100% (20000/20000)
{"appId":"0347313b-78cf-42dd-97a4-06d85bf00e83","refund_percentage":100,"refund_amount":20000,"fee_amount":0,"refund_status":"completed"}
[2026-03-21T01:50:32.281Z] [WARN] [refund:assert_failed] Refund assertion failed for app 184a59c0-3689-4b47-9acb-df912fde3d6c: refund_status is 'failed'
[2026-03-21T01:50:32.423Z] [WARN] [refund:assert_failed] Refund assertion failed for app b9a7ee00-86c1-435f-9b1e-3b604fd7d661: refund_status is 'failed'
[2026-03-21T01:50:32.595Z] [WARN] [refund:assert_failed] Refund assertion failed for app db786bda-4e4e-4a00-874d-dff744a10cb9: refund_status is 'failed'
[2026-03-21T01:50:32.751Z] [WARN] [refund:assert_failed] Refund assertion failed for app 1c600c8d-ebed-409a-9dc2-138829b253c1: refund_status is 'failed'
[2026-03-21T01:50:32.905Z] [WARN] [refund:assert_failed] Refund assertion failed for app d07a77a2-bf5c-410c-9f13-7095599cf37a: refund_status is 'failed'
[2026-03-21T01:50:33.059Z] [WARN] [refund:assert_failed] Refund assertion failed for app c32aaa86-6e57-45d4-bd8a-0a6ced5def2b: refund_status is 'failed'
[2026-03-21T01:50:33.215Z] [WARN] [refund:assert_failed] Refund assertion failed for app d5610966-7ff1-4c84-bc15-e2f17ba3b524: refund_status is 'failed'
[2026-03-21T01:50:33.361Z] [WARN] [refund:assert_failed] Refund assertion failed for app bc3cb0f7-6d8e-477c-bb52-2b07ed5f1cff: refund_status is 'failed'
[2026-03-21T01:50:33.507Z] [WARN] [refund:assert_failed] Refund assertion failed for app a33d1370-8f3c-4303-b696-c877a2aa9a36: refund_status is 'failed'
[2026-03-21T01:50:33.648Z] [WARN] [refund:assert_failed] Refund assertion failed for app cb28327f-5791-4423-abac-abb8a562fd33: refund_status is 'failed'
[2026-03-21T01:50:33.784Z] [WARN] [refund:assert_failed] Refund assertion failed for app 7260af17-a2d9-4292-a2b6-b6db7aafea84: refund_status is 'failed'
[2026-03-21T01:50:33.928Z] [WARN] [refund:assert_failed] Refund assertion failed for app b27cb7d7-f653-4b35-ae88-6e090b9493bf: refund_status is 'failed'
[2026-03-21T01:50:34.074Z] [WARN] [refund:assert_failed] Refund assertion failed for app 6170cfcf-d26b-4861-b60a-11dc430cc9cd: refund_status is 'failed'
[2026-03-21T01:50:34.232Z] [WARN] [refund:assert_failed] Refund assertion failed for app 52ef388b-5331-41d4-8485-84b53e99c886: refund_status is 'failed'
[2026-03-21T01:50:34.376Z] [WARN] [refund:assert_failed] Refund assertion failed for app 8ace8e2d-faf0-41a1-8e4f-660c3136f9b3: refund_status is 'failed'
[2026-03-21T01:50:34.527Z] [WARN] [refund:assert_failed] Refund assertion failed for app c55da832-3269-4444-963c-9ec383ee4034: refund_status is 'failed'
[2026-03-21T01:50:34.527Z] [INFO] [refund:done] Phase 4 complete: 9 refunded
{"refundedCount":9}
[2026-03-21T01:50:34.612Z] [INFO] [checkin:done] Event c3737d47-dbd4-499f-984d-6887847dab4b: 0 checked_in, 1 no_show
{"eventId":"c3737d47-dbd4-499f-984d-6887847dab4b","checkedIn":0,"noShow":1}
[2026-03-21T01:50:34.644Z] [INFO] [checkin:no_participants] No eligible participants for event 0bc94ce7-d4b2-43ed-b0b4-3941b79ade4a
[2026-03-21T01:50:34.754Z] [INFO] [checkin:done] Event 0b82a545-5823-4b6d-9b9c-aed49c13f3c1: 1 checked_in, 1 no_show
{"eventId":"0b82a545-5823-4b6d-9b9c-aed49c13f3c1","checkedIn":1,"noShow":1}
[2026-03-21T01:50:34.871Z] [INFO] [checkin:done] Event f09c933b-81f7-4d26-b6a4-56825336361f: 1 checked_in, 1 no_show
{"eventId":"f09c933b-81f7-4d26-b6a4-56825336361f","checkedIn":1,"noShow":1}
[2026-03-21T01:50:34.984Z] [INFO] [checkin:done] Event 7b09f85e-7899-41df-9849-8ca2c03024c3: 1 checked_in, 1 no_show
{"eventId":"7b09f85e-7899-41df-9849-8ca2c03024c3","checkedIn":1,"noShow":1}
[2026-03-21T01:50:35.037Z] [INFO] [checkin:no_participants] No eligible participants for event 14417c17-5d49-4c84-be20-6ffd13439ba2
[2026-03-21T01:50:35.064Z] [INFO] [checkin:no_participants] No eligible participants for event 69b09812-8a04-438f-964a-764a5183a012
[2026-03-21T01:50:35.181Z] [INFO] [checkin:done] Event 0a48c33e-fb7d-4708-9478-d576b81ddedf: 1 checked_in, 1 no_show
{"eventId":"0a48c33e-fb7d-4708-9478-d576b81ddedf","checkedIn":1,"noShow":1}
[2026-03-21T01:50:35.429Z] [INFO] [checkin:done] Event 490a8877-d369-47e7-b9e6-8974502c099f: 4 checked_in, 2 no_show
{"eventId":"490a8877-d369-47e7-b9e6-8974502c099f","checkedIn":4,"noShow":2}
[2026-03-21T01:50:35.672Z] [INFO] [checkin:done] Event 89fcc84c-630e-40ca-a936-dde448c43c63: 4 checked_in, 2 no_show
{"eventId":"89fcc84c-630e-40ca-a936-dde448c43c63","checkedIn":4,"noShow":2}
[2026-03-21T01:50:35.926Z] [INFO] [checkin:done] Event 1b575805-72b6-4835-8f3e-d13f61907d89: 4 checked_in, 2 no_show
{"eventId":"1b575805-72b6-4835-8f3e-d13f61907d89","checkedIn":4,"noShow":2}
[2026-03-21T01:50:36.149Z] [INFO] [checkin:done] Event 24ae1775-6420-4269-944e-4a802ce449de: 4 checked_in, 2 no_show
{"eventId":"24ae1775-6420-4269-944e-4a802ce449de","checkedIn":4,"noShow":2}
[2026-03-21T01:50:36.389Z] [INFO] [checkin:done] Event 7ad4da22-ff93-4307-bfb1-08aae6ef2048: 4 checked_in, 2 no_show
{"eventId":"7ad4da22-ff93-4307-bfb1-08aae6ef2048","checkedIn":4,"noShow":2}
[2026-03-21T01:50:36.675Z] [INFO] [checkin:done] Event 3cf9e99a-b799-4be1-9ee5-e73e49bfda14: 4 checked_in, 2 no_show
{"eventId":"3cf9e99a-b799-4be1-9ee5-e73e49bfda14","checkedIn":4,"noShow":2}
[2026-03-21T01:50:36.902Z] [INFO] [checkin:done] Event c18d4b29-4d86-439f-ac4e-23daf2324d19: 4 checked_in, 2 no_show
{"eventId":"c18d4b29-4d86-439f-ac4e-23daf2324d19","checkedIn":4,"noShow":2}
[2026-03-21T01:50:37.131Z] [INFO] [checkin:done] Event 1f3ad404-9826-433c-97ed-53775f2a3941: 4 checked_in, 2 no_show
{"eventId":"1f3ad404-9826-433c-97ed-53775f2a3941","checkedIn":4,"noShow":2}
[2026-03-21T01:50:37.363Z] [INFO] [checkin:done] Event 3200a102-66da-4e44-a245-9b2a93e744f7: 4 checked_in, 2 no_show
{"eventId":"3200a102-66da-4e44-a245-9b2a93e744f7","checkedIn":4,"noShow":2}
[2026-03-21T01:50:37.605Z] [INFO] [checkin:done] Event 3c2afcad-14ef-4276-a27a-92211ed52cbd: 4 checked_in, 2 no_show
{"eventId":"3c2afcad-14ef-4276-a27a-92211ed52cbd","checkedIn":4,"noShow":2}
[2026-03-21T01:50:37.829Z] [INFO] [checkin:done] Event 55848c6a-b35a-4269-97ef-955edb76e194: 4 checked_in, 2 no_show
{"eventId":"55848c6a-b35a-4269-97ef-955edb76e194","checkedIn":4,"noShow":2}
[2026-03-21T01:50:38.066Z] [INFO] [checkin:done] Event dcdfe0ab-8eab-41a5-a4ba-90f27c12b653: 4 checked_in, 2 no_show
{"eventId":"dcdfe0ab-8eab-41a5-a4ba-90f27c12b653","checkedIn":4,"noShow":2}
[2026-03-21T01:50:38.066Z] [INFO] [checkin:complete] Phase 5 checkin complete: 52 checked_in, 29 no_show
{"checkedInCount":52,"noShowCount":29}
[2026-03-21T01:50:38.213Z] [INFO] [match:insert_rules] Inserted basic match_rules for event c3737d47-dbd4-499f-984d-6887847dab4b
[2026-03-21T01:50:38.241Z] [ERROR] [match:fetch_checked_in] Failed to fetch checked_in participants for event c3737d47-dbd4-499f-984d-6887847dab4b: column event_participants.entry_group_id does not exist
[2026-03-21T01:50:38.355Z] [INFO] [match:insert_rules] Inserted basic match_rules for event 0bc94ce7-d4b2-43ed-b0b4-3941b79ade4a
[2026-03-21T01:50:38.384Z] [ERROR] [match:fetch_checked_in] Failed to fetch checked_in participants for event 0bc94ce7-d4b2-43ed-b0b4-3941b79ade4a: column event_participants.entry_group_id does not exist
[2026-03-21T01:50:38.504Z] [INFO] [match:insert_rules] Inserted basic match_rules for event 0b82a545-5823-4b6d-9b9c-aed49c13f3c1
[2026-03-21T01:50:38.532Z] [ERROR] [match:fetch_checked_in] Failed to fetch checked_in participants for event 0b82a545-5823-4b6d-9b9c-aed49c13f3c1: column event_participants.entry_group_id does not exist
[2026-03-21T01:50:38.643Z] [INFO] [match:insert_rules] Inserted basic match_rules for event f09c933b-81f7-4d26-b6a4-56825336361f
[2026-03-21T01:50:38.676Z] [ERROR] [match:fetch_checked_in] Failed to fetch checked_in participants for event f09c933b-81f7-4d26-b6a4-56825336361f: column event_participants.entry_group_id does not exist
[2026-03-21T01:50:38.808Z] [INFO] [match:insert_rules] Inserted basic match_rules for event 7b09f85e-7899-41df-9849-8ca2c03024c3
[2026-03-21T01:50:38.847Z] [ERROR] [match:fetch_checked_in] Failed to fetch checked_in participants for event 7b09f85e-7899-41df-9849-8ca2c03024c3: column event_participants.entry_group_id does not exist
[2026-03-21T01:50:38.966Z] [INFO] [match:insert_rules] Inserted basic match_rules for event 14417c17-5d49-4c84-be20-6ffd13439ba2
[2026-03-21T01:50:39.002Z] [ERROR] [match:fetch_checked_in] Failed to fetch checked_in participants for event 14417c17-5d49-4c84-be20-6ffd13439ba2: column event_participants.entry_group_id does not exist
[2026-03-21T01:50:39.121Z] [INFO] [match:insert_rules] Inserted basic match_rules for event 69b09812-8a04-438f-964a-764a5183a012
[2026-03-21T01:50:39.149Z] [ERROR] [match:fetch_checked_in] Failed to fetch checked_in participants for event 69b09812-8a04-438f-964a-764a5183a012: column event_participants.entry_group_id does not exist
[2026-03-21T01:50:39.272Z] [INFO] [match:insert_rules] Inserted basic match_rules for event 0a48c33e-fb7d-4708-9478-d576b81ddedf
[2026-03-21T01:50:39.301Z] [ERROR] [match:fetch_checked_in] Failed to fetch checked_in participants for event 0a48c33e-fb7d-4708-9478-d576b81ddedf: column event_participants.entry_group_id does not exist
[2026-03-21T01:50:39.420Z] [INFO] [match:insert_rules] Inserted basic match_rules for event 490a8877-d369-47e7-b9e6-8974502c099f
[2026-03-21T01:50:39.452Z] [ERROR] [match:fetch_checked_in] Failed to fetch checked_in participants for event 490a8877-d369-47e7-b9e6-8974502c099f: column event_participants.entry_group_id does not exist
[2026-03-21T01:50:39.577Z] [INFO] [match:insert_rules] Inserted basic match_rules for event 89fcc84c-630e-40ca-a936-dde448c43c63
[2026-03-21T01:50:39.606Z] [ERROR] [match:fetch_checked_in] Failed to fetch checked_in participants for event 89fcc84c-630e-40ca-a936-dde448c43c63: column event_participants.entry_group_id does not exist
[2026-03-21T01:50:39.719Z] [INFO] [match:insert_rules] Inserted basic match_rules for event 1b575805-72b6-4835-8f3e-d13f61907d89
[2026-03-21T01:50:39.747Z] [ERROR] [match:fetch_checked_in] Failed to fetch checked_in participants for event 1b575805-72b6-4835-8f3e-d13f61907d89: column event_participants.entry_group_id does not exist
[2026-03-21T01:50:39.859Z] [INFO] [match:insert_rules] Inserted basic match_rules for event 24ae1775-6420-4269-944e-4a802ce449de
[2026-03-21T01:50:39.885Z] [ERROR] [match:fetch_checked_in] Failed to fetch checked_in participants for event 24ae1775-6420-4269-944e-4a802ce449de: column event_participants.entry_group_id does not exist
[2026-03-21T01:50:39.998Z] [INFO] [match:insert_rules] Inserted basic match_rules for event 7ad4da22-ff93-4307-bfb1-08aae6ef2048
[2026-03-21T01:50:40.024Z] [ERROR] [match:fetch_checked_in] Failed to fetch checked_in participants for event 7ad4da22-ff93-4307-bfb1-08aae6ef2048: column event_participants.entry_group_id does not exist
[2026-03-21T01:50:40.141Z] [INFO] [match:insert_rules] Inserted basic match_rules for event 3cf9e99a-b799-4be1-9ee5-e73e49bfda14
[2026-03-21T01:50:40.169Z] [ERROR] [match:fetch_checked_in] Failed to fetch checked_in participants for event 3cf9e99a-b799-4be1-9ee5-e73e49bfda14: column event_participants.entry_group_id does not exist
[2026-03-21T01:50:40.289Z] [INFO] [match:insert_rules] Inserted basic match_rules for event c18d4b29-4d86-439f-ac4e-23daf2324d19
[2026-03-21T01:50:40.321Z] [ERROR] [match:fetch_checked_in] Failed to fetch checked_in participants for event c18d4b29-4d86-439f-ac4e-23daf2324d19: column event_participants.entry_group_id does not exist
[2026-03-21T01:50:40.433Z] [INFO] [match:insert_rules] Inserted basic match_rules for event 1f3ad404-9826-433c-97ed-53775f2a3941
[2026-03-21T01:50:40.461Z] [ERROR] [match:fetch_checked_in] Failed to fetch checked_in participants for event 1f3ad404-9826-433c-97ed-53775f2a3941: column event_participants.entry_group_id does not exist
[2026-03-21T01:50:40.573Z] [INFO] [match:insert_rules] Inserted basic match_rules for event 3200a102-66da-4e44-a245-9b2a93e744f7
[2026-03-21T01:50:40.601Z] [ERROR] [match:fetch_checked_in] Failed to fetch checked_in participants for event 3200a102-66da-4e44-a245-9b2a93e744f7: column event_participants.entry_group_id does not exist
[2026-03-21T01:50:40.712Z] [INFO] [match:insert_rules] Inserted basic match_rules for event 3c2afcad-14ef-4276-a27a-92211ed52cbd
[2026-03-21T01:50:40.740Z] [ERROR] [match:fetch_checked_in] Failed to fetch checked_in participants for event 3c2afcad-14ef-4276-a27a-92211ed52cbd: column event_participants.entry_group_id does not exist
[2026-03-21T01:50:40.857Z] [INFO] [match:insert_rules] Inserted basic match_rules for event 55848c6a-b35a-4269-97ef-955edb76e194
[2026-03-21T01:50:40.895Z] [ERROR] [match:fetch_checked_in] Failed to fetch checked_in participants for event 55848c6a-b35a-4269-97ef-955edb76e194: column event_participants.entry_group_id does not exist
[2026-03-21T01:50:41.007Z] [INFO] [match:insert_rules] Inserted basic match_rules for event dcdfe0ab-8eab-41a5-a4ba-90f27c12b653
[2026-03-21T01:50:41.033Z] [ERROR] [match:fetch_checked_in] Failed to fetch checked_in participants for event dcdfe0ab-8eab-41a5-a4ba-90f27c12b653: column event_participants.entry_group_id does not exist
[2026-03-21T01:50:41.033Z] [INFO] [match:complete] Phase 5 match complete: 0 pairs created
{"pairCount":0}
[2026-03-21T01:50:41.109Z] [INFO] [complete_events:completed] Event c3737d47-dbd4-499f-984d-6887847dab4b marked as completed
[2026-03-21T01:50:41.139Z] [INFO] [complete_events:completed] Event 0bc94ce7-d4b2-43ed-b0b4-3941b79ade4a marked as completed
[2026-03-21T01:50:41.174Z] [INFO] [complete_events:completed] Event 0b82a545-5823-4b6d-9b9c-aed49c13f3c1 marked as completed
[2026-03-21T01:50:41.206Z] [INFO] [complete_events:completed] Event f09c933b-81f7-4d26-b6a4-56825336361f marked as completed
[2026-03-21T01:50:41.237Z] [INFO] [complete_events:completed] Event 7b09f85e-7899-41df-9849-8ca2c03024c3 marked as completed
[2026-03-21T01:50:41.264Z] [INFO] [complete_events:completed] Event 14417c17-5d49-4c84-be20-6ffd13439ba2 marked as completed
[2026-03-21T01:50:41.293Z] [INFO] [complete_events:completed] Event 69b09812-8a04-438f-964a-764a5183a012 marked as completed
[2026-03-21T01:50:41.321Z] [INFO] [complete_events:completed] Event 0a48c33e-fb7d-4708-9478-d576b81ddedf marked as completed
[2026-03-21T01:50:41.351Z] [INFO] [complete_events:completed] Event 490a8877-d369-47e7-b9e6-8974502c099f marked as completed
[2026-03-21T01:50:41.379Z] [INFO] [complete_events:completed] Event 89fcc84c-630e-40ca-a936-dde448c43c63 marked as completed
[2026-03-21T01:50:41.409Z] [INFO] [complete_events:completed] Event 1b575805-72b6-4835-8f3e-d13f61907d89 marked as completed
[2026-03-21T01:50:41.444Z] [INFO] [complete_events:completed] Event 24ae1775-6420-4269-944e-4a802ce449de marked as completed
[2026-03-21T01:50:41.477Z] [INFO] [complete_events:completed] Event 7ad4da22-ff93-4307-bfb1-08aae6ef2048 marked as completed
[2026-03-21T01:50:41.506Z] [INFO] [complete_events:completed] Event 3cf9e99a-b799-4be1-9ee5-e73e49bfda14 marked as completed
[2026-03-21T01:50:41.539Z] [INFO] [complete_events:completed] Event c18d4b29-4d86-439f-ac4e-23daf2324d19 marked as completed
[2026-03-21T01:50:41.574Z] [INFO] [complete_events:completed] Event 1f3ad404-9826-433c-97ed-53775f2a3941 marked as completed
[2026-03-21T01:50:41.606Z] [INFO] [complete_events:completed] Event 3200a102-66da-4e44-a245-9b2a93e744f7 marked as completed
[2026-03-21T01:50:41.634Z] [INFO] [complete_events:completed] Event 3c2afcad-14ef-4276-a27a-92211ed52cbd marked as completed
[2026-03-21T01:50:41.665Z] [INFO] [complete_events:completed] Event 55848c6a-b35a-4269-97ef-955edb76e194 marked as completed
[2026-03-21T01:50:41.693Z] [INFO] [complete_events:completed] Event dcdfe0ab-8eab-41a5-a4ba-90f27c12b653 marked as completed
[2026-03-21T01:50:41.693Z] [INFO] [complete_events:done] Phase 5 complete_events done: 20 events completed
{"completedCount":20}
[2026-03-21T01:50:41.856Z] [INFO] [settle:verified] Settlement verified for event c3737d47-dbd4-499f-984d-6887847dab4b: id=3a9b6483-b771-4c95-bf57-ef64a9f21128, gross=0
{"eventId":"c3737d47-dbd4-499f-984d-6887847dab4b","settlementId":"3a9b6483-b771-4c95-bf57-ef64a9f21128","grossAmount":0}
[2026-03-21T01:50:42.023Z] [INFO] [settle:verified] Settlement verified for event 0bc94ce7-d4b2-43ed-b0b4-3941b79ade4a: id=28ce91d1-42e0-44fa-b237-797c8c94aa18, gross=0
{"eventId":"0bc94ce7-d4b2-43ed-b0b4-3941b79ade4a","settlementId":"28ce91d1-42e0-44fa-b237-797c8c94aa18","grossAmount":0}
[2026-03-21T01:50:42.160Z] [INFO] [settle:verified] Settlement verified for event 0b82a545-5823-4b6d-9b9c-aed49c13f3c1: id=b7eb340e-540c-410e-8613-0139ebd81b27, gross=0
{"eventId":"0b82a545-5823-4b6d-9b9c-aed49c13f3c1","settlementId":"b7eb340e-540c-410e-8613-0139ebd81b27","grossAmount":0}
[2026-03-21T01:50:42.314Z] [INFO] [settle:verified] Settlement verified for event f09c933b-81f7-4d26-b6a4-56825336361f: id=7d32d25f-223f-46bc-b8aa-9589272d11f1, gross=0
{"eventId":"f09c933b-81f7-4d26-b6a4-56825336361f","settlementId":"7d32d25f-223f-46bc-b8aa-9589272d11f1","grossAmount":0}
[2026-03-21T01:50:42.476Z] [INFO] [settle:verified] Settlement verified for event 7b09f85e-7899-41df-9849-8ca2c03024c3: id=ed632e1c-397d-478f-a45d-635ca673d656, gross=0
{"eventId":"7b09f85e-7899-41df-9849-8ca2c03024c3","settlementId":"ed632e1c-397d-478f-a45d-635ca673d656","grossAmount":0}
[2026-03-21T01:50:42.625Z] [INFO] [settle:verified] Settlement verified for event 14417c17-5d49-4c84-be20-6ffd13439ba2: id=1f232295-b8e8-4fcb-bd25-f3e894878bb8, gross=0
{"eventId":"14417c17-5d49-4c84-be20-6ffd13439ba2","settlementId":"1f232295-b8e8-4fcb-bd25-f3e894878bb8","grossAmount":0}
[2026-03-21T01:50:42.771Z] [INFO] [settle:verified] Settlement verified for event 69b09812-8a04-438f-964a-764a5183a012: id=1ce445bc-e151-446c-a8d2-b946c63f9f20, gross=0
{"eventId":"69b09812-8a04-438f-964a-764a5183a012","settlementId":"1ce445bc-e151-446c-a8d2-b946c63f9f20","grossAmount":0}
[2026-03-21T01:50:42.964Z] [INFO] [settle:verified] Settlement verified for event 0a48c33e-fb7d-4708-9478-d576b81ddedf: id=80a18789-1a5d-42be-a7cf-ec75d6cba20a, gross=0
{"eventId":"0a48c33e-fb7d-4708-9478-d576b81ddedf","settlementId":"80a18789-1a5d-42be-a7cf-ec75d6cba20a","grossAmount":0}
[2026-03-21T01:50:43.121Z] [INFO] [settle:verified] Settlement verified for event 490a8877-d369-47e7-b9e6-8974502c099f: id=f5548508-e7f8-4bf1-abf2-cb32f4509a22, gross=120000
{"eventId":"490a8877-d369-47e7-b9e6-8974502c099f","settlementId":"f5548508-e7f8-4bf1-abf2-cb32f4509a22","grossAmount":120000}
[2026-03-21T01:50:43.269Z] [INFO] [settle:verified] Settlement verified for event 89fcc84c-630e-40ca-a936-dde448c43c63: id=61f527c7-49a8-49a9-a573-5082c34a7736, gross=100000
{"eventId":"89fcc84c-630e-40ca-a936-dde448c43c63","settlementId":"61f527c7-49a8-49a9-a573-5082c34a7736","grossAmount":100000}
[2026-03-21T01:50:43.415Z] [INFO] [settle:verified] Settlement verified for event 1b575805-72b6-4835-8f3e-d13f61907d89: id=3bd97f2a-c79a-44c5-b078-6a8859acf95a, gross=100000
{"eventId":"1b575805-72b6-4835-8f3e-d13f61907d89","settlementId":"3bd97f2a-c79a-44c5-b078-6a8859acf95a","grossAmount":100000}
[2026-03-21T01:50:43.563Z] [INFO] [settle:verified] Settlement verified for event 24ae1775-6420-4269-944e-4a802ce449de: id=11c81ff2-029c-4872-9d5d-fdd6e0ff9901, gross=100000
{"eventId":"24ae1775-6420-4269-944e-4a802ce449de","settlementId":"11c81ff2-029c-4872-9d5d-fdd6e0ff9901","grossAmount":100000}
[2026-03-21T01:50:43.710Z] [INFO] [settle:verified] Settlement verified for event 7ad4da22-ff93-4307-bfb1-08aae6ef2048: id=ba7dd538-7050-4e45-ac8d-7407d4f4dbf9, gross=100000
{"eventId":"7ad4da22-ff93-4307-bfb1-08aae6ef2048","settlementId":"ba7dd538-7050-4e45-ac8d-7407d4f4dbf9","grossAmount":100000}
[2026-03-21T01:50:43.876Z] [INFO] [settle:verified] Settlement verified for event 3cf9e99a-b799-4be1-9ee5-e73e49bfda14: id=9487a528-ef6e-4096-b641-0b0cfc2050a3, gross=120000
{"eventId":"3cf9e99a-b799-4be1-9ee5-e73e49bfda14","settlementId":"9487a528-ef6e-4096-b641-0b0cfc2050a3","grossAmount":120000}
[2026-03-21T01:50:44.023Z] [INFO] [settle:verified] Settlement verified for event c18d4b29-4d86-439f-ac4e-23daf2324d19: id=b83b294b-ed54-45ae-8d60-b5c7fa72a273, gross=100000
{"eventId":"c18d4b29-4d86-439f-ac4e-23daf2324d19","settlementId":"b83b294b-ed54-45ae-8d60-b5c7fa72a273","grossAmount":100000}
[2026-03-21T01:50:44.175Z] [INFO] [settle:verified] Settlement verified for event 1f3ad404-9826-433c-97ed-53775f2a3941: id=dcfc43a5-6f39-4b5e-9f9b-ffc4623b5f3a, gross=120000
{"eventId":"1f3ad404-9826-433c-97ed-53775f2a3941","settlementId":"dcfc43a5-6f39-4b5e-9f9b-ffc4623b5f3a","grossAmount":120000}
[2026-03-21T01:50:44.347Z] [INFO] [settle:verified] Settlement verified for event 3200a102-66da-4e44-a245-9b2a93e744f7: id=61e5508c-036d-4832-ae79-7bc20e48d200, gross=100000
{"eventId":"3200a102-66da-4e44-a245-9b2a93e744f7","settlementId":"61e5508c-036d-4832-ae79-7bc20e48d200","grossAmount":100000}
[2026-03-21T01:50:44.512Z] [INFO] [settle:verified] Settlement verified for event 3c2afcad-14ef-4276-a27a-92211ed52cbd: id=46706218-4396-4b05-b0dd-ea01899545c5, gross=120000
{"eventId":"3c2afcad-14ef-4276-a27a-92211ed52cbd","settlementId":"46706218-4396-4b05-b0dd-ea01899545c5","grossAmount":120000}
[2026-03-21T01:50:44.656Z] [INFO] [settle:verified] Settlement verified for event 55848c6a-b35a-4269-97ef-955edb76e194: id=f6a1f359-576a-4795-aaf8-0714325ef117, gross=120000
{"eventId":"55848c6a-b35a-4269-97ef-955edb76e194","settlementId":"f6a1f359-576a-4795-aaf8-0714325ef117","grossAmount":120000}
[2026-03-21T01:50:44.813Z] [INFO] [settle:verified] Settlement verified for event dcdfe0ab-8eab-41a5-a4ba-90f27c12b653: id=9bf1bb2e-6a32-4c23-aca5-4e6116dfcdd9, gross=100000
{"eventId":"dcdfe0ab-8eab-41a5-a4ba-90f27c12b653","settlementId":"9bf1bb2e-6a32-4c23-aca5-4e6116dfcdd9","grossAmount":100000}
[2026-03-21T01:50:44.813Z] [INFO] [settle:complete] Phase 6 verify complete: 20 settlements verified
{"settlementCount":20}
[2026-03-21T01:50:44.933Z] [WARN] [settle_ready:still_pending] Settlement 3a9b6483-b771-4c95-bf57-ef64a9f21128 still not READY: status is 'PENDING'
[2026-03-21T01:50:45.020Z] [WARN] [settle_ready:still_pending] Settlement 28ce91d1-42e0-44fa-b237-797c8c94aa18 still not READY: status is 'PENDING'
[2026-03-21T01:50:45.101Z] [WARN] [settle_ready:still_pending] Settlement b7eb340e-540c-410e-8613-0139ebd81b27 still not READY: status is 'PENDING'
[2026-03-21T01:50:45.183Z] [WARN] [settle_ready:still_pending] Settlement 7d32d25f-223f-46bc-b8aa-9589272d11f1 still not READY: status is 'PENDING'
[2026-03-21T01:50:45.276Z] [WARN] [settle_ready:still_pending] Settlement ed632e1c-397d-478f-a45d-635ca673d656 still not READY: status is 'PENDING'
[2026-03-21T01:50:45.405Z] [WARN] [settle_ready:still_pending] Settlement 1f232295-b8e8-4fcb-bd25-f3e894878bb8 still not READY: status is 'PENDING'
[2026-03-21T01:50:45.493Z] [WARN] [settle_ready:still_pending] Settlement 1ce445bc-e151-446c-a8d2-b946c63f9f20 still not READY: status is 'PENDING'
[2026-03-21T01:50:45.598Z] [WARN] [settle_ready:still_pending] Settlement 80a18789-1a5d-42be-a7cf-ec75d6cba20a still not READY: status is 'PENDING'
[2026-03-21T01:50:45.685Z] [WARN] [settle_ready:still_pending] Settlement f5548508-e7f8-4bf1-abf2-cb32f4509a22 still not READY: status is 'PENDING'
[2026-03-21T01:50:45.773Z] [WARN] [settle_ready:still_pending] Settlement 61f527c7-49a8-49a9-a573-5082c34a7736 still not READY: status is 'PENDING'
[2026-03-21T01:50:45.876Z] [WARN] [settle_ready:still_pending] Settlement 3bd97f2a-c79a-44c5-b078-6a8859acf95a still not READY: status is 'PENDING'
[2026-03-21T01:50:45.961Z] [WARN] [settle_ready:still_pending] Settlement 11c81ff2-029c-4872-9d5d-fdd6e0ff9901 still not READY: status is 'PENDING'
[2026-03-21T01:50:46.049Z] [WARN] [settle_ready:still_pending] Settlement ba7dd538-7050-4e45-ac8d-7407d4f4dbf9 still not READY: status is 'PENDING'
[2026-03-21T01:50:46.140Z] [WARN] [settle_ready:still_pending] Settlement 9487a528-ef6e-4096-b641-0b0cfc2050a3 still not READY: status is 'PENDING'
[2026-03-21T01:50:46.226Z] [WARN] [settle_ready:still_pending] Settlement b83b294b-ed54-45ae-8d60-b5c7fa72a273 still not READY: status is 'PENDING'
[2026-03-21T01:50:46.328Z] [WARN] [settle_ready:still_pending] Settlement dcfc43a5-6f39-4b5e-9f9b-ffc4623b5f3a still not READY: status is 'PENDING'
[2026-03-21T01:50:46.428Z] [WARN] [settle_ready:still_pending] Settlement 61e5508c-036d-4832-ae79-7bc20e48d200 still not READY: status is 'PENDING'
[2026-03-21T01:50:46.527Z] [WARN] [settle_ready:still_pending] Settlement 46706218-4396-4b05-b0dd-ea01899545c5 still not READY: status is 'PENDING'
[2026-03-21T01:50:46.611Z] [WARN] [settle_ready:still_pending] Settlement f6a1f359-576a-4795-aaf8-0714325ef117 still not READY: status is 'PENDING'
[2026-03-21T01:50:46.719Z] [WARN] [settle_ready:still_pending] Settlement 9bf1bb2e-6a32-4c23-aca5-4e6116dfcdd9 still not READY: status is 'PENDING'
[2026-03-21T01:50:46.719Z] [INFO] [settle_ready:complete] Phase 6 ready transition complete: 0/20 transitioned
{"readyCount":0,"totalCount":20}
```

</details>

--- a/supabase/functions/backend-simulator/sim_assertions.ts
+++ b/supabase/functions/backend-simulator/sim_assertions.ts
@@ -143,9 +143,10 @@ export async function simAssertVerificationApproved(
       return fail("verification_approved", "approved", actualStatus, `submission status is '${actualStatus}'`);
     }
     // Also check that partner_verified_users row was created by the trigger
+    // Fix #181: partner_verified_users has composite PK (partner_id, user_id, verification_id), no 'id' column
     const { data: pvuData, error: pvuErr } = await supabase
       .from("partner_verified_users")
-      .select("id")
+      .select("submission_id")
       .eq("partner_id", data.partner_id)
       .eq("user_id", data.user_id)
       .eq("verification_id", data.verification_id)

--- a/supabase/functions/backend-simulator/sim_assertions_test.ts
+++ b/supabase/functions/backend-simulator/sim_assertions_test.ts
@@ -254,7 +254,7 @@ Deno.test("simAssertVerificationApproved - passes when approved + pvu exists", a
         }),
       },
       partner_verified_users: {
-        select: () => ({ data: { id: "pvu-1" }, error: null }),
+        select: () => ({ data: { submission_id: "pvu-1" }, error: null }),
       },
     },
   });

